### ISSUE TITLE
:paste <path>

### ISSUE THREAD

**Issue by zzzdeb** [ranger/ranger#1473]
Feature Request:
It would be nice if paste function take path as argument. For example I select multiple files and move them with short_cut f.e. with ',mD' to Documents. For now I achieve this behaviour through shell command. But then I lose features of ranger like task, status bar ...

**Comment by toonn** [comment]
Mark some files, using space, then `yy` to yank them or `dd` to cut them, move to `~/Documents` and then `pp` to paste them.

**Comment by zzzdeb** [comment]
1. that is exactrly the reason why I wish this feature. For every frequently used dirs&files, I have shortcuts to do basic actions like opening, opening in new tab, moving or coping files to them. Making this actions doesnt need to make me leave current dir to come back after paste, I think. I just want to do this with one shortcut. I achieved with chain command better result. Only it erases my last visited dir also cant use '' after this action. Still feels hacky for basic action.
```
map Ya chain copy; cd  ~/hiwi/ACPLT-DevKit-linux64/acplt; paste; cd -
map ma chain cut; cd  ~/hiwi/ACPLT-DevKit-linux64/acplt; paste; cd -
```
2. For paste function to be able to paste only in current dir seems to be too harsh restriction.
3. As I see in source code
``` python
class CopyLoader
def __init__
self.original_path = self.fm.thistab.path
```
defines where to paste. Seems easily modifieble.

**Comment by toonn** [comment]
Hmm, seems like you're just looking for the `:mv` or `:cp` commands?

**Comment by zzzdeb** [comment]
exactly

**Comment by toonn** [comment]
👍

**Comment by zzzdeb** [comment]
I am new in github. I dont understand what this means. Does it mean that was it?

**Comment by toonn** [comment]
@zzzdeb I ask whether `:mv` or `:cp` was what you wanted and you said yes...

You want to do `:paste %s /to/some/path`, right? That sounds like `:mv %s /to/some/path` would replace your cut/paste and `:cp ...` would replace your copy/paste.

**Comment by zzzdeb** [comment]
mv cp functions implemented? I couldnt find them and understand what :move does. No documentation for :move.  Should I open another issue for these functions?

**Comment by toonn** [comment]
Oh, you're right, i apologize. I assumed we had wrappers for them since we do for some other thing like `:cd` and `:touch`, etc.

I suggest either creating custom commands or just mapping something along these lines: `map pm console shell mv %s%space` or `map pc console shell cp %c%space`.

The first one would move the marked files to a directory of your choosing. The latter would copy the files in the copybuffer to the directory of your choosing. With `mv` you might wanna add `chain console shell mv...; redraw_window`.

**Comment by zzzdeb** [comment]
First I did it like you explained. But there was few things I didnt like. 
1. I lose ranger features like task, status, which are really nice.
2. If the command failes there is no notification in ranger, I couldnt be sure if command was executed, Always having to check was cumbersome.
3. For large files ranger gets stuck, till command is done.
So I used chain. With above chain command, I achieved pretty much what I wanted except losing last visited dir. I thought this features (mv, cp) could be ranger default features.
